src/metrics/mod.rs

```diff
@@ -38,6 +42,29 @@ pub static LRU_CACHE_MISS: Lazy<Family<KindLabel, Counter>> = Lazy::new(|| {
     metric
 });
 
+pub static RPC_METHOD_FAILURE: Lazy<Family<RpcMethodLabel, Counter>> = Lazy::new(|| {
+    let metric = Family::default();
+    DEFAULT_REGISTRY.write().register(
+        "rpc_method_failure",
+        "Number of failed RPC calls",
+        metric.clone(),
+    );
+    metric
+});
+
+pub static RPC_METHOD_TIME: Lazy<Family<RpcMethodLabel, Histogram>> = Lazy::new(|| {
+    let metric = Family::<RpcMethodLabel, Histogram>::new_with_constructor(|| {
+        Histogram::new(exponential_buckets(0.01, 2.0, 10))
+    });
```

Comment:
What do these numbers represent?

Reply:
Added a comment. Bucket sizes are from 10ms to 5.12s, each bucket is twice as big the previous.